assets/css/style.css

```diff
@@ -64,6 +64,27 @@ html[data-theme="dark"] {
   --col-white: #fff;
 }
 
+/*ScrollBar*/
```

Comment:
Put a comment in the style.css where the relevant toggle button code begins

``` terminal
/* 
---------------------------------
         SCROLL BAR
----------------------------------
*/
```
so that the file is more readable